testing/collectdtesting/fauxllectd.py

```diff
@@ -0,0 +1,64 @@
+"""To be imported as collectd module for unit testing (conftest.py) and development.
```

Comment:
See note below about how this is only for integration testing.  This should go in the utils dir if it needs to directly interact with collectd plugins.